int8-model/benchmark-unet.sh

```diff
@@ -4,23 +4,34 @@
 
 set -xeu
 
-if (( $# != 1 && $# != 2 )); then
-  echo "usage: $0 <hip-device-id> [<ipra-path-prefix>]"
+if (( $# != 2 && $# != 3 )); then
+  echo "usage: $0 <hip-device-id> <batch-size> [<ipra-path-prefix>]"
   exit 1
 fi
 
-IRPA_PATH_PREFIX="${2:-/data/shark}"
+readonly SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)"
+readonly IREE_BENCHMARK="$(which iree-benchmark-module)"
+readonly HIP_DEVICE="$1"
+readonly BATCH_SIZE="$2"
+if ! [[ "${BATCH_SIZE}" =~ ^(1|4|8|14)$ ]]; then
+  echo "Allowed batch-sizes: 1, 4, 8, 14"
+  exit 1
+fi
+INPUT_PATH="${SCRIPT_DIR}/unet_npys"
+INPUTS="--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/latent_model_input.npy \
+--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/guidance_scale.npy \
+--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/prompt_embeds.npy \
+--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/add_text_embeds.npy \
+--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/add_time_ids.npy \
+--input=@${INPUT_PATH}/unet_inputs_bs${BATCH_SIZE}/t.npy"
```

Comment:
Add quotes around paths, otherwise this will break if we check out the repo in a path with spaces

Reply:
Oh sorry, I missed that